mypy2junit.py

```diff
@@ -57,13 +58,34 @@ def process_lines(lines: List[str]) -> Tuple[str, bool]:
     return output, int(result['count']) == 0
 
 
+parser = argparse.ArgumentParser()
+parser.add_argument('files', metavar='FILE', nargs='*', help='files to read, if empty, stdin is used')
+parser.add_argument('--output',
+                    type=str,
+                    dest='output',
+                    help='Filename to output to')
+parser.add_argument('--tee', action='store_true')
```

Comment:
I think it would be better to define this inside the main function, not at the module (constants) level.